Python/bytecodes.c

```diff
@@ -4009,25 +4031,24 @@ dummy_func(
             DEOPT_IF(callable_o != (PyObject *)&PyType_Type);
         }
 
-        op(_CALL_TYPE_1, (callable, null, arg -- res)) {
+        op(_CALL_TYPE_1, (callable, null, arg -- res, c, a)) {
             PyObject *arg_o = PyStackRef_AsPyObjectBorrow(arg);
-
             assert(oparg == 1);
-            DEAD(null);
-            DEAD(callable);
-            (void)callable; // Silence compiler warnings about unused variables
-            (void)null;
             STAT_INC(CALL, hit);
             res = PyStackRef_FromPyObjectNew(Py_TYPE(arg_o));
-            PyStackRef_CLOSE(arg);
+            INPUTS_DEAD();
+            c = callable;
+            a = arg;
         }
 
         macro(CALL_TYPE_1) =
             unused/1 +
             unused/2 +
             _GUARD_NOS_NULL +
             _GUARD_CALLABLE_TYPE_1 +
-            _CALL_TYPE_1;
+            _CALL_TYPE_1 +
+            POP_TOP +
```

Comment:
This can be a `_POP_TOP_NOP` as `type` is immortal. You can add a comment there explaining this beside the uop append.